app/models/user/reputation_tokens/automation_feedback_author_token.rb

```diff
@@ -11,5 +11,5 @@ class User::ReputationTokens::AutomationFeedbackAuthorToken < User::ReputationTo
 
   def i18n_params =  { exercise_title: representation.exercise.title }
   def guard_params = "Representation##{representation.id}"
-  def internal_url = Exercism::Routes.edit_mentoring_automation_path(representation)
+  def internal_url = Exercism::Routes.edit_mentoring_automation_path(representation.uuid)
```

Comment:
The idomatic way to do this is to add a `to_param` method to the representation instead and leave these as they were (much more DRY).

```ruby
def to_param
  uuid
end
```